### PR TITLE
feat: add divider component

### PR DESCRIPTION
**Pull request by nmerget**

### CODE REVIEW

**Review by mfranzke — COMMENTED** [review]
How are those tab changes related to the divider component ?

File: packages/components/_templates/mitosis/new/model.ejs.t [hunk 1]
```diff
@@ -2,14 +2,15 @@
 to: src/components/<%= name %>/model.ts
 ---
 export interface DB<%= h.changeCase.pascal(name) %>DefaultProps {
+	className?:string;
 }
 
 export interface DB<%= h.changeCase.pascal(name) %>WcProps {
-  stylePath?: string;
+  	stylePath?: string;
```
Comment (annsch) [inline packages/components/_templates/mitosis/new/model.ejs.t:9]: is this blank correct?

File: packages/components/_templates/mitosis/new/model.ejs.t [hunk 2]
```diff
@@ -2,14 +2,15 @@
...
 }
 
 export type DB<%= h.changeCase.pascal(name) %>Props = DB<%= h.changeCase.pascal(name) %>DefaultProps & DB<%= h.changeCase.pascal(name) %>WcProps;
 
 export interface DB<%= h.changeCase.pascal(name) %>State {
-  stylePath?: string;
+  	stylePath?: string;
```
Comment (annsch) [inline packages/components/_templates/mitosis/new/model.ejs.t:15]: same here

File: packages/components/src/components/divider/divider.scss [hunk 3]
```diff
@@ -0,0 +1,24 @@
+@use "@db-ui/foundations/build/scss/variables" as *;
+@use "@db-ui/foundations/build/scss/variables.global" as *;
+@use "@db-ui/foundations/build/scss/helpers/functions" as *;
+
+.db-divider {
+	height: to-rem(1);
+	width: 100%;
+
+	opacity: 0.1;
+	background: $db-colors-secondary-enabled;
```
Comment (annsch) [inline packages/components/src/components/divider/divider.scss:10]: shouldn't this change on different backgrounds?

File: packages/components/_templates/mitosis/new/model.ejs.t (hunk 1 above)
Comment (nmerget) [reply packages/components/_templates/mitosis/new/model.ejs.t:9]: https://github.com/db-ui/mono/blob/main/.editorconfig#L11

Yes @mfranzke changed it

File: packages/components/_templates/mitosis/new/model.ejs.t (hunk 2 above)
Comment (nmerget) [reply packages/components/_templates/mitosis/new/model.ejs.t:15]: https://github.com/db-ui/mono/pull/10#discussion_r1034726405

File: packages/components/src/components/divider/divider.scss (hunk 3 above)
Comment (nmerget) [reply packages/components/src/components/divider/divider.scss]: fixed

**Review by annsch — APPROVED** [review]
LGTM

File: packages/components/scripts/watch-filter.js [hunk 4]
```diff
@@ -0,0 +1,6 @@
+module.exports = (file) => {
+	if (file.endsWith('.css')) {
+		console.log(file);
+	}
+	return !file.endsWith('.css');
+};
```
Comment (mfranzke) [inline packages/components/scripts/watch-filter.js:6]: @nmerget where is this file being in use? Could you please add a comment that documents / includes some information?

File: showcases/react-showcase/src/index.scss [hunk 5]
```diff
@@ -1,3 +1,16 @@
...
+
+.icon {
+	margin: auto 0;
+}
+
+.test-div {
+	display: flex;
+	height: $db-sizing-md;
+	width: $db-sizing-lg;
+}
```
Comment (mfranzke) [inline showcases/react-showcase/src/index.scss:16]: Is this still in use somewhere ?

Comment (mfranzke) [inline showcases/react-showcase/src/index.scss:16]: Is this still in use somewhere ?

**Pull request merged**